pom.xml

```diff
@@ -105,6 +105,40 @@
             <version>1.9.5</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins.workflow</groupId>
```

Comment:
Plugin wouldn't work without workflow deps. That's terrible that in jenkins now impossible to cloud and other plugins without installing workflow.